src/app/shared/components/workshop-card/workshop-card.component.ts

```diff
@@ -74,6 +81,7 @@ export class WorkshopCardComponent implements OnInit, OnDestroy {
     private imagesService: ImagesService) { }
 
   ngOnInit(): void {
+    this.isPrivate = this.workshopData.providerOwnership === 'Private' && this.userRoleView === Role.provider;
```

Comment:
1) this.workshopData.providerOwnership  !== 'Common'
2) this.userRoleView === Role.provider; is redundant because this check will be in the template
3) availableSeats !== WORKSHOP_UNLIMITED_SEATS

Reply:
fixed